.github/workflows/push-docker-images.yml

```diff
@@ -0,0 +1,49 @@
+name: Build and Push Docker Images
+
+on:
+  push:
+    branches:
+      - master
+    tags:
+      - open-release/*
+jobs:
+  push:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      # Use the release name as the image tag if we're building an open release tag.
+      # Examples: if we're building 'open-release/maple.1', tag the image as 'maple.1'.
+      # Otherwise, we must be building from a push to master, so use 'latest'.
+      - name: Get tag name
+        id: get-tag-name
+        uses: actions/github-script@v5
+        with:
+          script: |
+            const releasePrefix = 'refs/tags/open-release/';
+            const tagName = context.ref.split(releasePrefix)[1] || 'latest';
+            console.log('Will use tag: ' + tagName);
+            return tagName;
+          result-encoding: string
+
+      - name: Build and push Dev Docker image
+        uses: docker/build-push-action@v1
+        with:
+          push: true
+          username: ${{ secrets.DOCKERHUB_USERNAME }}
+          password: ${{ secrets.DOCKERHUB_PASSWORD }}
+          target: dev
+          repository: edxops/analytics-api-dev
+          tags: ${{ steps.get-tag-name.outputs.result }},${{ github.sha }}
+
+      # - name: Build and push prod Docker image
```

Comment:
Why is this commented out? Is it the case we don't really push this image to any repositories?

Reply:
I have commented that out on purpose. Since our initial target is `devstack` so I am just pushing the dev images for now. Also the old Ansible based `devstack` image is pushed to `edxops/analytics_api` repository and enabling this push will override that image too. For smooth transition, I am pushing the Native Images to new repository(`analytics_api_dev`) and keeping the old image intact so if we want to switch over to older image due to some issues we always can.

Reply:
If you are doing this on purpose, then the whole section should be wrapped with comments to help others understand why this is left in the code, but commented out. Please provide the explanation in the code to help others understand the desire for the smooth transition.